_docs/reference/config/mixer/mixer-config.md

```diff
@@ -80,8 +80,8 @@ params:
     value: response.time  # from the set of canonical attributes
     labels:
       srcConsumerId: source.user | source.uid
-      targetResponseStatusCode: response.code
-      targetServiceName: target.service
+      destinationResponseStatusCode: response.code
+      destinationServiceName: destination.service
 ```
```

Comment:
In mixer/v1/config/cfg.proto these are:
```
//       src_consumer_id: source.user | source.uid
//       dst_response_status_code: response.code
//       dst_service_name: destination.service
```
What are the correct names? If the long name is used for the destination ones, should `srcConsumerId` be changed to `sourceConsumerId" for consistency?

Reply:
good catch! @ZackButcher / @mandarjog / @douglas-reid please clarify.

Reply:
I didn't bother spending a lot of time with the mixer config proto updates since, as Doug said, we need to do a pass on all of those anyway. Everything in `_docs/reference` is going to be regenerated soon anyway.